courses-app/src/app/services/spinner.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@angular/core';
+import { Subject } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class SpinnerService {
+
+  public spinnerObservable = new Subject<boolean>();
```

Comment:
Better to create `'private'` property - it helps to prevent to change it without methods.